nixos/modules/services/misc/nitter.nix

```diff
@@ -45,6 +45,11 @@ let
   '';
 in
 {
+  imports = [
+    # https://github.com/zedeus/nitter/pull/772
```

Comment:
```suggestion
```
is this already removed in the package

Reply:
Yes, currently this option does nothing in NixPkgs unstable.